tests/unit/draw/DefaultDrawer.spec.js

```diff
@@ -31,40 +32,164 @@ describe('Test Class: DefaultDrawer()', () => {
         exit: jest.fn(() => drawer.d3),
         remove: jest.fn(() => drawer.d3),
       };
+      drawer.initializeComponents = jest.fn();
       drawer.drawDefaultResource = jest.fn();
     });
 
-    it('Draw should call drawDefaultResource method', () => {
+    it('Draw should call the internal methods', () => {
       drawer.draw([]);
 
+      expect(drawer.initializeComponents).toBeCalled();
       expect(drawer.drawDefaultResource).toBeCalled();
     });
   });
 
-  describe('Test method: drawDefaultResource', () => {
+  describe('Test method: initializeComponents', () => {
+    let drawer;
+    let component01;
+    let component02;
+    let svgList;
+
+    beforeEach(() => {
+      document.body.innerHTML = '<svg id="viewport"></svg>';
```

Comment:
not sure it's needed

Reply:
We need it because the tests call for it and it can't be empty.